pkgs/development/tools/misc/dura/default.nix

```
@@ -0,0 +1,39 @@
+{ lib, fetchFromGitHub, rustPlatform, pkgs }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "dura";
+  version = "a41863879772650f2538a978ca926357bae674db";
```

```suggestion
  version = "unstable-2022-01-07";
```

Per the [Nixpkgs naming conventions](https://nixos.org/manual/nixpkgs/stable/#sec-package-naming) (also in the commit message)